scripts/azureml-assets/azureml/assets/validate_assets.py

```diff
@@ -227,11 +227,27 @@ def validate_name(asset_config: assets.AssetConfig) -> int:
     return error_count
 
 
+def validate_categories(asset_config: assets.AssetConfig) -> int:
+    """Validate asset categories.
+
+    Args:
+        asset_config (AssetConfig): Asset config.
+
+    Returns:
+        int: Number of errors.
+    """
+    if len(asset_config.categories) == 0:
+        _log_error(asset_config, "Categories not found")
+
+    return len(asset_config.categories) == 0

```

Comment:
Maybe a little simpler:
```suggestion
    if not asset_config.categories:
        _log_error(asset_config, "Categories not found")
        return 1
    return 0
```